### ISSUE TITLE
find the estimated time travel time of different mode 

### ISSUE THREAD

**Issue by JYSW380** [jaychenblue/scout-concordia#218]
It will indicate the estimated travel time to get to the destination with the current selected mode of transportation.

This is related to #25 

Here is the checklist of features that should be included for this issue to be considered completed:
[ ] toast the estimated time at bottom of the app and the UI to be confirmed.

once these features have been completed, this issue can be moved to the "In Review" board.

**Comment by jaychenblue** [comment]
@JYSW380 Is there a mockup corresponding to this issue? If so, please attach it to this page.

**Comment by JYSW380** [comment]
> @JYSW380 Is there a mockup corresponding to this issue? If so, please attach it to this page.

we don't have a mockup so far, this issue will be mainly focus on the backend how to make the estimated time logic.

**Comment by jaychenblue** [comment]
This issue should include the front end too. I think it may depend on #206.

**Comment by jaychenblue** [comment]
According to @JenYn, #206 won't be implemented.

@JYSW380 What does the UI for this issue currently look like? Can you post a screenshot?

**Comment by JYSW380** [comment]
It will be looks like this:
<img width="392" alt="Screen Shot 2020-04-06 at 6 25 30 PM" src="https://user-images.githubusercontent.com/41170611/78611263-91ebbe80-7834-11ea-92b3-a2d0df1e0934.png">

**Comment by JYSW380** [comment]
@jaychenblue @JenYn @vincentcerri @ManpreetSohal @Averynder 
I pushed some commits on branch estimate-travel-time branch, and I am able to find building to building locations and estimated time.
But there are still 2 things left: 
1. classroom to classroom locations (the classroom should located in different buildings, like H-101 to CC-100, do not need the estimated time that 2 classrooms located in the same building).
2.right now the displacement for showing the the locations is in the middle of the app, it should be displaced just above the "schedule map shuttle bus" bar.(like screenshot shown above).

**Comment by jaychenblue** [comment]
@JYSW380 The first thing is up to you. You can either hard code an estimated time or calculate based on the path of nodes. @JenYn, what do you think?

Is this issue related to #156?

**Comment by JYSW380** [comment]
yes, it is related to #156, after @vicmanea finishing #156, they can be integrated.
